invenio_app_rdm/theme/views.py

```diff
@@ -29,6 +30,20 @@ def create_url_rule(rule, default_view_func):
         return {"rule": rule, "view_func": default_view_func}
 
 
+def _can_manage_community():
+    """Function used to check if a user has permissions to manage a community."""
+    return administration_permission.can() and request.url_rule.rule.startswith(
+        "/communities/"
+    )
+
+
+def _can_manage_record():
+    """Function used to check if a user has permissions to manage a record."""
+    return administration_permission.can() and request.url_rule.rule.startswith(
+        "/records/"
```

Comment:
this will add it to search pages as well if I am not mistaken? is this what we want?

Reply:
It should only be displayed when we have the context of a record, which if I am not mistaken happens all the time under `/records/`. I couldn't find any search page that starts with `/records/`.
The general search starts with `/search` and the community records search starts with `/communities/`.
If you find any please share it and I will think of alternatives.